src/state/selectors/windows.js

```diff
@@ -112,3 +113,21 @@ export const getCompanionAreaVisibility = createSelector(
     return !!(companionAreaOpen && sideBarOpen);
   },
 );
+
+/**
+ * Returns the draggability of a window
+ * @param {object} state
+ * @param {object} props
+ * @return {Boolean}
+ */
+export const getWindowDraggability = createSelector(
```

Comment:
Folks have been saying a lot of words about selectors lately.  Not sure if this is the correct usage or not.